Lib/asyncio/tasks.py

```diff
@@ -461,6 +464,27 @@ def _on_completion(f):
     return done, pending
 
 
+async def _cancel_and_wait(fut, loop):
+    """Cancel the *fut* future or task and wait until it completes."""
+
+    waiter = loop.create_future()
+    cb = functools.partial(_release_waiter, waiter)
+    fut.add_done_callback(cb)
+    fut.cancel()
+
+    try:
+        # We cannot wait on *fut* directly to make
+        # sure _cancel_and_wait itself is reliably cancellable.
+        await waiter
+    except futures.CancelledError:
```

Comment:
Do we need this `except` at all?

Reply:
No, it's an oversight from an earlier code iteration.